### PR TITLE
Add clear links for forgot password, change password

### PR DESCRIPTION
**Pull request by gregelin**
Added clearer links to take user to "forgot password" page and "change password" page.

### CODE REVIEW

File: templates/navbar.html [hunk 1]
```diff
@@ -98,6 +98,7 @@
               </li>
               {% endif %}
               <li role="separator" class="divider"></li>
+              <li><a href="/accounts/password/change/">Change Password</a>{{environment.}}</li>
```
Comment (davidpofo) [inline templates/navbar.html:101]: Is that period in environment supposed to be there? Also where does that variable come from? @gregelin

Comment (gregelin) [reply templates/navbar.html:101]: I think `{{environment.}}` is accidental debug/dev code I failed to delete.

Comment (davidpofo) [reply templates/navbar.html:101]: okay i'll delete it directly.

**Review by davidpofo — APPROVED** [review]
lgtm pr party 4/22

**Pull request merged**